jsonschema/_utils.py

```diff
@@ -38,6 +38,22 @@ def __repr__(self):
         return repr(self.store)
 
 
+class Cache(object):
```

Comment:
Ah yes, good point. I was considering just making the maxsize a parameter, but making the decorator function itself a parameter makes sense.